ktor-server/jvm/src/io/ktor/server/application/plugins/api/PluginPhases.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.application.plugins.api
+
+import io.ktor.server.application.*
+
+/**
+ * Callable object that provides handlers to extend handling of an application call.
+ **/
```

Comment:
handlers to extend handling feels like a tongue-twister somewhat and not entirely clear.